www/js/control/ControlDataTable.jsx

```diff
@@ -6,11 +6,12 @@ import { array } from "prop-types";
 // Note the camelCase to dash-case conventions when translating to .html files!
 // val with explicit call toString() to resolve bool values not showing
 const ControlDataTable = ({ controlData }) => {
-  console.log("Printing data trying to tabulate", controlData);
+  // console.log("Printing data trying to tabulate", controlData);
   return (
+    //rows require unique keys!
     <DataTable style={styles.table}>
       {controlData.map((e) =>
-        <DataTable.Row>
+        <DataTable.Row key={e.key}> 
```

Comment:
At this point, we expect at most one notification per day so this should be fine. Feel free to resolve once you have seen this.